docs/en/enterprise-edition/policy-reference/ibm-policies/ibm-iam-policies/bc-ibm-2-4.adoc

```diff
@@ -0,0 +1,49 @@
+== IBM Cloud Multi-Factor Authentication (MFA) not enabled at the account level in Terraform
+
+=== Policy Details
+
+[width=45%]
+[cols="1,1"]
+|===
+|Prisma Cloud Policy ID
+| TBD
+
+|Checkov ID
+| https://github.com/bridgecrewio/checkov/blob/main/checkov/terraform/checks/graph_checks/ibm/IBM_EnableMFAatAccountLevel.yaml[CKV2_IBM_4]
+
+|Severity
+|MEDIUM
+
+|Subtype
+|Build
+
+|Frameworks
+|Terraform,TerraformPlan
+
+|===
+
+=== Description
+
+This policy is focused on ensuring that Multi-Factor Authentication (MFA) is enabled at the account level within an IAM (Identity and Access Management) framework. MFA requires users to provide multiple forms of verification before granting access, typically a combination of something they know (like a password) and something they have (like a smartphone or security token).
+
+The primary reason for this policy is to enhance security. Enabling MFA greatly reduces the risk of unauthorized access even if a password is compromised. Having MFA in place mitigates potential security breaches, protecting sensitive data and resources from being accessed by malicious actors. Not enabling MFA leaves the system vulnerable to various attacks such as phishing, credential stuffing, and brute force attacks, thereby posing significant security threats.
+
```

Comment:
This policy detects whether Multi-Factor Authentication (MFA) is disabled at the account level within an Identity and Access Management (IAM) framework. Disabled MFA increases vulnerability to attacks such as phishing, credential stuffing, and brute force. MFA requires users to provide multiple forms of verification, helping to protect sensitive data and resources by preventing unauthorized access, even if passwords are compromised.

@tsmithv11  - This suggestion replaces lines 27-29.

Reply:
Looks good. Thanks!